fffw/graph/meta.py

```diff
@@ -346,6 +348,11 @@ def validate(self) -> None:
         else:
             assert str(self.dar) == 'nan'
 
+        if self.duration != 0:
+            assert abs(self.frames / self.duration - self.frame_rate) <= 0.001
+        else:
+            assert self.frames == 0
```

Comment:
?
```suggestion
            assert self.frames == 0
            assert self.frame_ratу == 0
```

Reply:
Entered `start` parameter into game, because stream duration includes pause before first frame, if present (`Trim` filter makes one and `SetPTS` eliminates it)